tests/functional/d/distutil_regression_import_error.py

```diff
@@ -0,0 +1,13 @@
+"""
+Regression test to check that distutils can be imported
```

Comment:
```suggestion
"""Regression test to check that distutils can be imported
```

We need to find some pre-commit hook that starts to enforce this.
Perhaps we can look at something for `Path` at the same time? 😄

Reply:
Maybe https://github.com/PyCQA/pydocstyle ?

Reply:
Better yet https://github.com/myint/docformatter

Reply:
For module docstrings I think both are fine. Those are tests after all.

> Perhaps we can look at something for Path at the same time? 😄

I wouldn't recommend that. It's not really worth it and might limit us more than it actually would help.

Reply:
> I wouldn't recommend that. It's not really worth it and might limit us more than it actually would help.

This was a little joke as @Pierre-Sassoulas keeps having to remind me to use it on the PRs I open 😄 
Don't think we should enforce it all the time indeed.

Perhaps those packages allow us to exclude modules? Oh well, something to explore another time.

Reply:
Yes we definitely don't want to refactor all the old code that uses os.path.